File: Gravstark.rs
fn main() {
    let name = "@GravStark";
    let nationality = "Italian";
    let age = 18;
    let city = "Modena";

    let interests = vec!["Programming", "VideoGames", "Music", "Healthcare"];
    let languages = vec!["Italian", "English", "Finnish"];
    let programming_languages = vec!["C", "C++", "C#", "Batch", "Assembly", "Java", "Html", "CSS", "JavaScript"];

    println!("Hi, I'm {}, an {} {} y/o student from {}.", name, nationality, age, city);
    println!("✨ Interests:");
    for interest in interests.iter() {
        println!("- {}", interest);
    }
    println!("✨ Languages:");
    for language in languages.iter() {
        println!("- {}", language);
    }
    println!("✨ Programming languages:");
    for programming_language in programming_languages.iter() {
        println!("- {}", programming_language);
    }
}

File: GravStark.rb
name = "@GravStark"
nationality = "Italian"
age = 18
city = "Modena"

interests = ["Programming", "VideoGames", "Music", "Healthcare"]
languages = ["Italian", "English", "Finnish"]
programming_languages = ["C", "C++", "C#", "Batch", "Assembly", "Java", "Html", "CSS", "JavaScript"]

puts "Hi, I'm #{name}, an #{nationality} #{age} y/o student from #{city}."
puts "✨ Interests:"
interests.each { |interest| puts "- #{interest}" }
puts "✨ Languages:"
languages.each { |language| puts "- #{language}" }
puts "✨ Programming languages:"
programming_languages.each { |programming_language| puts "- #{programming_language}" }
